tests/integration/evpn.es/01-esi-lag.yml

```diff
@@ -0,0 +1,94 @@
+message: |
+  The devices under test are an ESI-LAG pair connected to an Arista EOS switch. The
+  Arista EOS switch should have a working LAG, and the attached hosts should be
+  able to ping each other.
+
+plugin: [ 'evpn.es' ]
+
+defaults.vxlan.start_vni: 20000
+bgp.as: 65000
+
+evpn.ethernet_segments:
+  seg_1.auto: true
+
+groups:
+  _auto_create: true
+  switches:
+    members: [ s1, s2 ]
+    module: [ vlan, vxlan, ospf, bgp, evpn, lag ]
+    evpn._junos_default_macvrf: true  # let's keep it simple for now
+  probes:
+    members: [ x1 ]
+    module: [ lag, vlan ]
+    device: eos
+  hosts:
+    members: [ h1, h2, h3 ]
+    device: linux
+    provider: clab
+
+vlans:
+  red:
+    mode: bridge
+    links: [ h1-x1, h2-s1, h3-s2 ]
+
+links:
+# EVPN/VXLAN Link
+- s1:
+  s2:
+  mtu: 1600
+# ESI-LAG
+- lag:
+    members:
+    - s1:
+        lag.lacp_system_id: 1
```

Comment:
As commented above, I think it would be great if the plugin could implement logic to auto-generate the lacp_system_id:
When a lag interface references an ES with 'auto: true', if there is no manual override generate a lacp_system_id based on the minimum node ID of all the nodes attached to that segment (could be >2)